nifi-commons/nifi-status-history/src/main/java/org/apache/nifi/controller/status/history/VolatileComponentStatusRepository.java

```diff
@@ -248,6 +250,26 @@ public StatusHistory getNodeStatusHistory(final Date start, final Date end) {
         return new StandardStatusHistory(snapshots, new HashMap<>(), new Date());
     }
 
+    @Override
+    public Collection<String> getProcessGroupIds() {
```

Comment:
I would consider one of the following implementation options:

- Returning the actual ids
- Throwing UnsupportedOperationException exception